### PR TITLE
Added paulstretch cask

### PR DESCRIPTION
**Pull request by ojczeo**
### Adding paulstretch app for mac os x
https://github.com/paulnasca/paulstretch_cpp/issues/9
http://hypermammut.sourceforge.net/paulstretch/ cask

### CODE REVIEW

**Comment by esaruoho** [comment]
wouldbe lovely if this could be gotten to homebrew cask, easier to install for sure :)

File: Casks/paulstretch.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'paulstretch' do
+  version '2.2.2'
+  sha256 '9f23fe69965ab479e250c9705adaa8f01ea6fc50c3e5ff83f6a949d03a86a21b'
+
+  url "http://www.cornwarning.com/xfer/PaulStretch-#{version}-OSX-10.6.dmg"
+  appcast 'https://github.com/paulnasca/paulstretch_cpp/releases.atom'
```
Comment (vitorgalvao) [inline Casks/paulstretch.rb]: There aren’t any releases. This `appcast` does nothing.

File: Casks/paulstretch.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  name 'PaulStretch'
+  homepage 'http://hypermammut.sourceforge.net/paulstretch/'
+
+  app 'PaulStretch/paulstretch.app'
+
+end
```
Comment (vitorgalvao) [inline Casks/paulstretch.rb]: It has a “no newline at end of file” warning. Those are [actually meaningful](http://unix.stackexchange.com/a/18789).

File: Casks/paulstretch.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  version '2.2.2'
+  sha256 '9f23fe69965ab479e250c9705adaa8f01ea6fc50c3e5ff83f6a949d03a86a21b'
+
+  url "http://www.cornwarning.com/xfer/PaulStretch-#{version}-OSX-10.6.dmg"
+  appcast 'https://github.com/paulnasca/paulstretch_cpp/releases.atom'
+  name 'PaulStretch'
+  homepage 'http://hypermammut.sourceforge.net/paulstretch/'
+
+  app 'PaulStretch/paulstretch.app'
+
```
Comment (vitorgalvao) [inline Casks/paulstretch.rb]: Extraneous line.

**Review by vitorgalvao — COMMENTED** [review]
Welcome and thank you, @ojczeo.

Please try again and **please and do not delete the template**. Readd it and follow the steps.

**Pull request closed without merging**